### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Jordiet71** [Jarnsen/ha-xsense-component_test#5] [General]
Thanks, seems to work well following your instructions. Device SC07-WX

**Comment by Jarnsen** [comment]
thats sound good.

**Comment by shantonio** [comment]
Thanks first of all . 
Here is my positiv feedback on tested new Hydrometer  **STH0A** with small display   (FW v1.1.0)
Seems to work well following your instructions.

<img width="2904" height="574" alt="image" src="https://github.com/user-attachments/assets/97a5322f-22bf-4e62-99a5-1ecde40a29a4" />
